tests/e2e/kepler_internal_test.go

```diff
@@ -54,19 +73,114 @@ func TestKeplerInternal_Reconciliation(t *testing.T) {
 	assert.Equal(t, 1, len(containers))
 	assert.Equal(t, 1, len(containers[0].Ports))
 	assert.EqualValues(t, 9108, containers[0].Ports[0].ContainerPort)
+	// test expected status
+	testKeplerInternalStatus(t, f, ki)
```

Comment:
@sunya-ch , could you please add the assertions to framework instead? `f.AssertInternalStatus`

Reply:
Move to the assertions.go: https://github.com/sustainable-computing-io/kepler-operator/compare/e75fcca04c20ae4911abdedbedd383b6c6bd76aa..3238a90270873cf411818cae59f0b41e505b74c3